### PR TITLE
Update Credits Page

### LINKED ISSUE #4060: Update credits page
https://scratch.mit.edu/credits should have the following updates:

**Add to MIT Scratch Team section:**

Amielle (originalwow)
Craig (noncanonical)
Joshua (Class12321)
Ellen (SunnyDay4aBlueJay)

**Remove from MIT Scratch Team section:**

Elizabeth (rmiel)

**Add to Past Contributors:**

Elizabeth Foster

**Remove from Past Contributors:**

Ellen Daoust

## files to change

* src/views/credits/people.json
* src/views/credits/credits.jsx

### PR DESCRIPTION
**Pull request by GrahamSH-LLK**
### Resolves:

Resolves #4060 

### Changes:

Changed everything in #4060 

@benjiwheeler

### CODE REVIEW

File: src/views/credits/people.json [hunk 1]
```diff
@@ -53,6 +58,11 @@
...
         "userId": 10866958,
         "name": "Colby"
+    },
+     {
+        "userName": "noncanonical",
+        "userId": 55851826,
+        "name": "Craig"
     },
     {
         "userName": "harakou",
```
Comment (R4356th) [inline src/views/credits/people.json:68]: It should be Harakou. But it is simple though.

Comment (GrahamSH-LLK) [reply src/views/credits/people.json]: Got it! I fixed it!

**Comment by GrahamSH-LLK** [comment]
@benjiwheeler I _think_ i have been pinging you, but I wasn't sure, so I tried again. In case I haven't been, I finished.

File: src/views/credits/people.json (hunk 1 above)
Comment (R4356th) [reply src/views/credits/people.json]: Great! It was not a bug produced by you though.

Comment (GrahamSH-LLK) [reply src/views/credits/people.json]: Oh well.

**Comment by GrahamSH-LLK** [comment]
@benjiwheeler Have you seen this yet?

**Comment by IamTregsthedev** [comment]
> @benjiwheeler Have you seen this yet?

Wait some time I guess, He'll Notice it :)

**Comment by benjiwheeler** [comment]
sorry for the delay, @GrahamSH-LLK! Reviewing it now!

**Comment by benjiwheeler** [comment]
Thanks @GrahamSH-LLK , and thanks for catching the capitalization of Harakou's username!

**Pull request merged**